make-it/EmMakefile

```diff
@@ -92,11 +92,11 @@ ifeq ($(TARGET), asmjs-unknown-emscripten)
 endif
 
 ifeq ($(TARGET), wasm32-unknown-emscripten)
-   TARGET_FLAG = -s WASM=1 -s BINARYEN_TRAP_MODE=js -s ALLOW_MEMORY_GROWTH=1
```

Comment:
Looks like [BINARYEN_TRAP_MODE has been removed](https://github.com/emscripten-core/emscripten/issues/10365#issuecomment-856810416). As long as the [Double.Test.js tests](https://github.com/ni/VireoSDK/blob/main/test-it/karma/publicapi/Double.Test.js) pass I think we are good, they were the [reason we initially had the flag](https://github.com/ni/VireoSDK/commit/790d42a5de70b1b08f99c4a3b7c7d620ddf659d8).

Reply:
Now because of the default clang trap mode, we are facing test failure in Double.Test.js as you mentioned. 
So we added an extra function in [WriteDoubleToMemory ](https://github.com/ni/VireoSDK/blob/c8eb11d197b09f7ed2100bdb01cd48cd8f2fa003/source/core/TypeAndDataManager.cpp#L2847). This function will handle overflow and underflow from double to Int as JS trap was doing. 

@rajsite  Is this an intended change or do you have any other comment?